src/Tickets/Commerce/Shortcodes/Checkout_Shortcode.php

```diff
@@ -124,6 +124,7 @@ public function get_gateways_connected() {
 	 */
 	public static function enqueue_assets() {
 		// Enqueue assets.
+		do_action( 'tec-tickets-commerce-checkout-shortcode-assets' );
```

Comment:
I'm curious why are we adding a `do_action` here?

Reply:
By removing the asset from the group and adding an action to load the assets instead, the 'conditionals' param is called.

Reply:
Thanks for the clarification. Yep, I understand that. I'm curious why is that we're having the new action there, instead of using `tribe_asset_enqueue()`.

Reply:
Because we are loading (possibly) several assets at once. In the future, when/if we add more gateways, having this hook will allow the gateways to tie into the hook easily, rather than us manually calling an asset for each gateway and having to update the code each time.

Reply:
Makes sense!